rts/lra/lra-service-base/src/main/java/io/narayana/lra/LRAData.java

```diff
@@ -8,14 +8,16 @@
  * for JSON response creation when LRA info is asked for.
  */
 public class LRAData {
-    private final String lraId;
-    private final String clientId;
-    private final LRAStatus status;
-    private final boolean isTopLevel;
-    private final boolean isRecovering;
-    private final long startTime;
-    private final long finishTime;
-    private final int httpStatus;
+    private String lraId;
```

Comment:
I don't understand the purpose of the change here. The `LRAData` was intentionally made immutable. Why is needed this to be changed?

Reply:
Used for the JSON (fasterxml) exchanges. FasterXML works with a default constructor and setters/getters still :)

Reply:
Ok, I see. I was thinking there will be some reason just I haven't got it from the context. Then this is fine.